Packs/AtlassianConfluenceCloud/pack_metadata.json

```diff
@@ -2,20 +2,21 @@
     "name": "Atlassian Confluence Cloud",
     "description": "Atlassian Confluence Cloud allows users to interact with confluence entities like content, space, users and groups. Users can also manage the space permissions.",
     "support": "xsoar",
-    "currentVersion": "1.0.25",
+    "currentVersion": "1.1.0",
     "author": "Cortex XSOAR",
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
     "created": "2021-08-16T11:16:19Z",
     "categories": [
-        "IT Services"
```

Comment:
why this change? do all collectors have this category?
seems like we should have both if anything

Reply:
[PA103] Fails when having both. 
![image](https://github.com/user-attachments/assets/644c08a8-a881-4ac9-86a1-7f6c7a3aacff)

Reply:
done